# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/wasm/wasm-resizablearraybuffer-shared.js
# Commit: 75d829afb4b0
# Full Hash: 75d829afb4b06a3a8b7e91ea15c8a0cf34665fb7
# Author: Cosmin Sabou <csabou@mozilla.com>
# Date: 2025-09-16 12:02 +0300
# Description:
#   Revert "Bug 1969509 - wasm: Update .maxByteLength to report the source wasm max pages in bytes. r=rhunt" for causing javascript engine crashes (bug 1988694)
# ==============================================================================

diff -r 896c8ee9171d -r 75d829afb4b0 js/src/jit-test/tests/wasm/wasm-resizablearraybuffer-shared.js
--- a/js/src/jit-test/tests/wasm/wasm-resizablearraybuffer-shared.js	Tue Sep 16 04:43:56 2025 +0300
+++ b/js/src/jit-test/tests/wasm/wasm-resizablearraybuffer-shared.js	Tue Sep 16 12:02:13 2025 +0300
@@ -89,7 +89,7 @@
 assertEq(ab2.growable, false);
 assertEq(mem.buffer, ab2);
 assertEq(ab2 !== ab, true);
-assertEq(ab2.maxByteLength, 50 << 16);
+assertEq(ab2.maxByteLength, 20 << 16);
 
 assertEq(ins.exports.check(0, 10, 3), 1);
 ins.exports.fill(0, 10, 2);